config_src/infra/FMS2/MOM_io_infra.F90

```diff
@@ -666,19 +674,22 @@ subroutine get_axis_data( axis, dat )
   type(axistype),     intent(in)  :: axis !< An axis type
   real, dimension(:), intent(out) :: dat  !< The data in the axis variable
 
-  integer :: i
-
-  ! This routine might not be needed for MOM6.
-  if (allocated(axis%ax_data)) then
-    if (size(axis%ax_data) > size(dat)) call MOM_error(FATAL, &
-      "get_axis_data called with too small of an output data array for "//trim(axis%name))
-    do i=1,size(axis%ax_data) ; dat(i) = axis%ax_data(i) ; enddo
-  elseif (.not.FMS2_writes) then
-    call mpp_get_axis_data( axis%AT, dat )
-  endif
+  call mpp_get_axis_data( axis%AT, dat )
 
 end subroutine get_axis_data
 
+
+!> Extracts and returns the axis data stored in an axistype.
```

Comment:
This does not seem like the right description for the subroutine `set_axis_data()`.  Rather it seems to have been copied unmodified from `get_axis_data()`.  Please correct this desciption.